app/assets/javascripts/sufia/save_work/save_work_control.es6

```diff
@@ -95,6 +95,19 @@ export class SaveWorkControl {
   validateFiles() {
     if (!this.isNew || this.uploads.hasFiles) {
       this.requiredFiles.check()
+
+      if (this.uploads.hasFiles) {
+        // TODO: A flag to track that the user has agreed during this session should
+        //       be set when the user clicks on the deposit agreement checkbox.
+        let agreedRealtime = true
+        if (agreedRealtime) {
+          return true
+        }
+        if (this.uploads.agreedLasttime) {
```

Comment:
Correct, that's why I have the TODO in the comments. I've been manually setting this value as I run the code to test the logic.